### PR TITLE
Chevron with coupler support

### PR DESCRIPTION
**Pull request by Jacfomg**
Using https://github.com/qiboteam/qibolab/pull/508 and https://github.com/qiboteam/qibolab_platforms_qrc/pull/49 for the couplers this add the needed coupler pulse. Also, it won't break non-coupler platforms.

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.
- [ ] Compatibility with Qibo modules (Please edit this section if the current pull request is not compatible with the following branches).
    - [ ] Qibo: `master`
    - [ ] Qibolab:  https://github.com/qiboteam/qibolab/pull/508
    - [ ] Qibolab_platforms_qrc: `main`

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -16,7 +16,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
-qibolab = { git = "https://github.com/qiboteam/qibolab.git", branch = "main" }
+qibolab = { git = "https://github.com/qiboteam/qibolab.git", branch = "dummy2-now-with-couplers!" }
 qibo = "^0.1.9"
```
Comment (andrea-pasquale) [inline pyproject.toml:20]:
```suggestion
qibo = "^0.2.0"
```
This should fix tests @Jacfomg

Comment (andrea-pasquale) [reply pyproject.toml]: Nevermind :(

**Comment by andrea-pasquale** [comment]
The fix for this has been implemented already in #501 and #406

https://github.com/qiboteam/qibocal/blob/666f533d61d69c8026ae8132f8abc0dcad5a42be/tests/test_randomized_benchmarking.py#L171

I believe that both should be merged soon

**Review by andrea-pasquale — CHANGES_REQUESTED** [review]
I tested the chevron using qblox and it raised the following error:

```sh
  return self.qubits[qubit].native_gates.RX.pulse(start, relative_phase)
AttributeError: 'NoneType' object has no attribute 'pulse'
```
Probably there is something wrong in the corresponding PR in qibolab.

**Comment by Jacfomg** [comment]
> I tested the chevron using qblox and it raised the following error:
> 
> 
> 
> ```sh
> 
>   return self.qubits[qubit].native_gates.RX.pulse(start, relative_phase)
> 
> AttributeError: 'NoneType' object has no attribute 'pulse'
> 
> ```
> 
> Probably there is something wrong in the corresponding PR in qibolab.

Okay, I will take a look

**Comment by Jacfomg** [comment]
@andrea-pasquale take a look when the asking 2q gate routines to act on `qubits: [2,3]` or `qubits: [3,2]`. I did some reordering for them to pass tests but maybe we want something different.

**Comment by Jacfomg** [comment]
If cov gets fixed this should be ready now

**Review by andrea-pasquale — APPROVED** [review]
Thanks @Jacfomg

**Review by Edoardo-Pedicillo — APPROVED** [review]

**Pull request merged**